workshops/templates/workshops/event.html

```diff
@@ -6,6 +6,17 @@
 {% load tags %}
 
 {% block content %}
+
+{% if event.tags_changed %}
```

Comment:
So is it just looking for changes to tags (as the name suggests) or to 
changes to any field?